.circleci/config.yml

```diff
@@ -50,7 +50,8 @@ variables:
       name: Run tests
       command: |
         mkdir test-reports
-        py.test --cov=kipoi/ tests/ -n 4 --no-cov-on-fail --junitxml=test-reports/junit.xml
+        #py.test --cov=kipoi/ tests/ -n 1 --no-cov-on-fail --junitxml=test-reports/junit.xml
+        py.test --cov=kipoi/ tests/ --verbose
```

Comment:
Does that work for reporting the coverage?

Reply:
nope,  this is this is still from trying a left-over to debug the circle-ci issues, since I discovered those isses here (#428),
I'll continue on this PR sonish